app/_hub/kong-inc/jq/index.md

```diff
@@ -0,0 +1,96 @@
+---
+name: Kong jq
+publisher: Kong Inc.
+versions: 0.1.x
+beta: true
+
+desc: Provides arbitrary jq transformations to a JSON object.
+description: |
+    Provides arbitrary jq transformations to a JSON object. The source of the object can either be the request or response body, and the transformed result can either replace the body or be used to set HTTP headers.
+
+    Multiple filters can be specified in the configuration. You can use these filters to set request headers from values present in the request body, and then rewrite the response body with a separate jq program.
+
+enterprise: true
+type: plugin
+categories:
+ - transformations
+
+kong_version_compatibility:
+    enterprise_edition:
+      compatible: 
+        - 2.6.x
+
+params:
+  name: jq
+  service_id: true
+  route_id: true
+  consumer_id: true
+  yaml_examples: false
+  k8s_examples: false
+  konnect_examples: false
+  protocols: ["http"]
+  dbless_compatible:
+  config:
+    - name: request_jq_program
+      required: semi
+      datatype: string
+      description: |
+        The jq program to run on the request body. For example, `.[0] | { "X-Foo": .foo }`. 
+        Either `request_jq_program` or `response_jq_plugin` must be included in configuration.
+    - name: request_jq_options
```

Comment:
It looks like the name of the parameter is request_jq_program_options. cc @pintsized or @bungle for confirmation

Reply:
yeah I was going on the readme, then looked at the schema but for the wrong branch - after switching to the master branch I see it has changed - will update